test/api/v3/unit/models/group.test.js

```diff
@@ -607,6 +613,90 @@ describe('Group Model', () => {
       });
     });
 
+    describe('#checkChatSpam', () => {
+      let testMessage, testUser, testTime;
+      let testUserID = '1';
+      beforeEach(async () => {
+        party._id = TAVERN_ID;
+        testTime = Date.now();
+        testMessage = {
```

Comment:
Might be better to create a function that returns a test message object. Something like:

```js
function generateTestMessage (overrides = {}) {
  return Object.assign({}, overrides, {
    text: 'test message',
    uuid: testUserID,
    timestamp: testTime,
  });
}
```

That way, you create a new object every time, not reference the same object over and over.